setup.py

```diff
@@ -49,16 +49,15 @@
     author_email='pipermerriam@gmail.com',
     url='https://github.com/ethereum/eth-tester',
     include_package_data=True,
-    install_requires=[
-        "toolz>0.8.2,<1;implementation_name=='pypy'",
-        "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
+    install_requires=[    
+        "eth-abi>=1.0.0,<2",
```

Comment:
Removed allowing betas as 1.0.0 came out:
https://github.com/ethereum/eth-abi/releases

Still getting this
```
eth-utils 1.2.2 has requirement eth-typing<2.0.0,>=1.3.0, but you'll have eth-typing 2.0.0 which is incompatible.
py-evm 0.2.0a33 has requirement eth-typing<2.0.0,>=1.1.0, but you'll have eth-typing 2.0.0 which is incompatible.
```
due to the eth-abi mention [here](https://github.com/ethereum/web3.py/issues/1056#issuecomment-428420981).

Reply:
@stefanmendoza if you look through the full `pip install` logs there should be a line which indicates where/why `eth-typing==2.0.0` was chosen.

If that doesn't lead to a fix, the ultimate fix for this is to upgrade both `eth-utils` and `py-evm` to newer versions which support `eth-typing` (@carver we may need to cut a new `py-evm` beta for that since the newer `eth-typing` support was just recently merged).

Reply:
I'm kind of confused by whatcha mean?

I think it's coming from eth-abi because in the dependencies ([gist here]()), `eth-abi` is the only thing allow 2.x:
```
eth-tester==0.1.0b33
  - eth-abi [required: >=1.0.0-beta.1r,<2, installed: 1.2.0]
    - eth-typing [required: <=2, installed: 2.0.0]
```
I ran into a similar situation in https://github.com/ethereum/web3.py/issues/1056#issuecomment-428420981, but realized @carver had already fixed this in [1.x](https://github.com/ethereum/eth-abi/pull/100) and [2.x](https://github.com/ethereum/eth-abi/pull/99) - patch versions just haven't been released yet. Once eth-abi-1.2.1 is released, this should be fixed automatically by pulling in the latest version on `pip install`, right?

Reply:
I'll get some eth-abi releases out.